README.md

```diff
@@ -204,8 +231,66 @@ other log management system if you wish to persist logs.  See the
 [signalfx-agent command](./docs/signalfx-agent.1.man) doc for more information on
 supported command flags.
 
+#### Windows Standalone .zip
+If you don't want to use the installer script, we offer a
+.zip that can be deployed to the target host.  This bundle is available for
+download on the [Github Releases
+Page](https://github.com/signalfx/signalfx-agent/releases) for each new
+release.
+
+Before proceeding please satisfy the following three requirements are installed
+on the target host.
+* [Microsoft Visual C++ 2008 SP1 Redistributable Package (x64)](https://www.microsoft.com/en-us/download/details.aspx?id=2092)
+* [.Net Framework 3.5](https://docs.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows-10) (Windows 8+)
+* [Visual C++ Compiler for Python 2.7](https://www.microsoft.com/EN-US/DOWNLOAD/DETAILS.ASPX?ID=44266)
+
+To use the bundle:
+
+1) Unzip it to a directory of your choice on the target system.
+
+2) Ensure a valid configuration file is available somewhere on the target
+system.  The main thing that the installer script provides -- but that you will
+have to provide manually with the bundle -- is a run directory for the agent to
+use.  Since you aren't installing from a package, there are three config
+options that you will especially want to consider:
+
+ - `internalStatusHost` - This is the host name that
+	 the agent will listen on so that the `signalfx-agent status` command can
+	 read diagnostic information from a running agent.  This is also the host name the
+	 agent will listen on to serve internal metrics about the agent.  These metrics can
+	 can be scraped by the `internal-metrics` monitor.  This will default to `localhost`
+	 if left blank.
+
+ - `internalStatusPort` - This is the port that the agent will listen on so that
+	 the `signalfx-agent status` command can read diagnostic information from
+	 a running agent.  This is also the host name the agent will listen on to serve
+	 internal metrics about the agent.  These metrics can can be scraped by the
+	 `internal-metrics` monitor.  This will default to `8095`.
+
+See the section on [Privileges](#privileges) for information on the
+capabilities the agent requires.
+
+3) Run the agent by invoking the agent executable
+`SignalFxAgent\bin\signalfx-agent.exe -config <path to config.yaml>`.  The agent
+logs only to stdout/err so it is up to you to direct that to a log file or
+other log management system if you wish to persist logs.  See the
+[signalfx-agent command](./docs/signalfx-agent.1.man) doc for more information on
+supported command flags.
+
+4) You may optionally install the agent as a windows service by invoking the
+agent executable and specifying a few command line flags
```

Comment:
Sentence needs a period. Also, "a few command line flags" is pretty vague. Do you list them somewhere else? How would they know which flags to specify?